### PR TITLE
Milestone 1 

### PR DESCRIPTION
**Pull request by t-bello7**
- Create Database schema in schema.sql file
- Insert Data into the table created from the schema in data.sql file
- Query table in queries.sql file

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name varchar(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL,
 );
```
Comment (mateo951) [inline schema.sql:10]: Well done! You created a table called `animals` with required columns ✔

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon',  TO_DATE('03/02/2020', 'DD/MM/YYYY'), 0, TRUE, 10.23 );
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 2, TRUE, 8);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', TO_DATE('07/01/2021', 'DD/MM/YYYY'), 1, FALSE, 15.04);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', TO_DATE('12/05/2017', 'DD/MM/YYYY'), 5, TRUE, 11 );
```
Comment (mateo951) [inline data.sql:6]: Awesome 🔥 You introduced the propper data 🧐

**Review by mateo951 — APPROVED** [review]
### Hi @t-bello7, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 😎 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (t-bello7) [reply data.sql:6]: Way to go Mateo. Thanks for reviewing my code 👾

**Pull request merged**